### PR TITLE
Page 4 feature

### PR DESCRIPTION
**Pull request by bauHau**
This feature adds the page_4.html to the application.

### CODE REVIEW

File: page_4.html [hunk 1]
```diff
@@ -29,6 +29,10 @@ <h1 class="text-center">Page 4</h1>
 			proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
 		</section>
 
+		<section>
```
Comment (fullStackDevs) [inline page_4.html]: The content of this section should be put inside a container.

File: page_4.html [hunk 2]
```diff
@@ -29,6 +29,10 @@ <h1 class="text-center">Page 4</h1>
...
+			<h3>Section two</h3>
+		</section>
```
Comment (fullStackDevs) [inline page_4.html]: The content of this section should be put inside a container.

File: page_4.html [hunk 3]
```diff
@@ -31,6 +31,12 @@ <h1 class="text-center">Page 4</h1>
 
 		<section>
 			<h3>Section two</h3>
```
Comment (fullStackDevs) [inline page_4.html]: This section content should also be put inside a container.

File: page_4.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+		<!--[if lt IE 9]>
+			<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.2/html5shiv.min.js"></script>
+			<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+		<![endif]-->
+	</head>
+	<body>
+		<h1 class="text-center">Page 4</h1>
+
+		<section>
+			<h3>Section one</h3>
```
Comment (fullStackDevs) [inline page_4.html]: The content of this section should be put inside a container.

File: page_4.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+			tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+			quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+			consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+			cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+			proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+		</section>
+
+		<section>
+			<h3>Section two</h3>
```
Comment (fullStackDevs) [inline page_4.html]: The content of this section should be also put inside a container.

**Review by fullStackDevs — CHANGES_REQUESTED** [review]
The section included in this page should be put inside containers for proper alignment.

**Review by bauHau — COMMENTED** [review]
This looks ok.

**Pull request merged**